docs/reference/nodes/iterator.md

```diff
@@ -0,0 +1,47 @@
+# Iteration Nodes
+
+Iteration Nodes are used to create and define loops within a workflow. 
+
+## Iterator Node
+The *Iterator* node is used to create a loop in a workflow. This node generates multiple execution subgraphs with the nodes following the *Iterator* node. Each execution of the loop uses one value from the input iterable object as if it was effectively iterating over its elements.
+
+![Iterator Node](../../images/nodes/iterator.png) 
+
+>[!NOTE]
+> *Iterator* Nodes can be nested, meaning that another *Iterator* node can be placed inside the loop of an *Iterator* node to achieve multiple levels of iteration.
+
+### Inputs
+
+- **inValues** : This plug of type [Object](../nodes/#plugs) expects an iterable Python object, such as lists, dictionaries, sets or tuples. The number of elements of the iterable object will determine the number of loops the *Iterator* node will execute.
+
+### Outputs
+- **outValue**: This plug of type [Object](../nodes/#plugs) will output one value of the input iterable object corresponding to the current iteration. This means that the output of this plug will change dynamically for each loop the iteration Node executes. 
+
+## IteratorEnd Node
+The *IteratorEnd* node is used to define the end of a loop started by an *Iterator* node. This means that all nodes between an *Iterator* and a *IteratorEnd* will be part of the execution loop. It is recommended to always have an *IteratorEnd* node per *Iterator* node in a workflow.
+
+![IteratorEnd Node](../../images/nodes/iteratorEnd.png)  
+
+>[!NOTE]
+> It is advised to connect to the *IteratorEnd* node the output generated in the loop via a custom `through` plug. 
+
+## ListAccumulator Node

```

Comment:
List accumulator uses an instance of the list instead of a copy of the list object, and sets the result to the input as well as the output.  Basically, in each iteration, the node updates its list input. 
Given that ListAppend outputs an object copy, I understand that what you had to do is update a global variable with the output of the ListAppend. That wouldn't be needed with the ListAccumulator node.

Reply:
Ok, so the difference would be this one?
![image](https://github.com/user-attachments/assets/7b987fd2-b4f7-49b8-8a5f-242419b66642)
I'm trying to understand it to know if the documentation it's ok, because I see it a littlebit difficult to imagine how to use.

Reply:
You wouldn't necessarily need to connect the list variable to the list accumulator. The default value of the inList plug is already a list.

Reply:
Just to add, I belive ListAccumulator nodes were developed a lot before workflow variables so what you are doing here with Variable nodes was possible before only using ListAccumulators.

Reply:
Ok I see. Before you could not save values, I sse the needed of this before the variables. nothing to modify then. Thx!